backend/scripts/transfer-contract-ownership.ts

```diff
@@ -1,37 +1,117 @@
-import { SafeBulkWriter } from 'shared/safe-bulk-writer'
+import { createSupabaseDirectClient, pgp } from 'shared/supabase/init'
+
 import { runScript } from './run-script'
-import { convertUser } from 'common/supabase/users'
+import yargs from 'yargs'
+import { hideBin } from 'yargs/helpers'
+
+interface Args {
+  slugs: string[]
+  toUserId: string
+}
+
+interface User {
+  id: string
+  username: string
+  name: string
+  created_time: string
+}
+
+interface Contract {
+  id: string
+  data: any
+}
+
+const supabaseDirectClient = createSupabaseDirectClient()
```

Comment:
This initializes before the secret keys are loaded from runScript. You are supposed to use the pg passed from runscript:
`runScript(async ({ pg }) => {`

Reply:
+1